cli/options/options.go

```diff
@@ -198,6 +199,9 @@ func GetConfigFromContext(ctx *cli.Context) (config.Config, error) {
 		configFile   = ctx.String("config-file")
 		relativePath = ctx.String("relative-path")
 	)
+	if ctx.String("config-file") == "" && ctx.String("config-path") == "" {
+		return config2.VmConfig, nil
+	}
```

Comment:
It's not that easy, we use GetConfigFromContext not only for VM but also for other `node` commands.

Reply:
We probably need an additional argument passed to `GetConfigFromContext`